src/android/CHIPTest/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# For more information about using CMake with Android Studio, read the
```

Comment:
I write this file by hand because I need -whole-archive one utest lib

        -Wl,-whole-archive
        ${MATTER_SDK_BUILD}/lib/${MATTER_SDK_UTEST_LIB}
        -Wl,-no-whole-archive
        
The MATTER_SDK_UTEST_LIB is libPlatformTests.a or other library by build args, I did not find anyway way to do it in gn, so can not generated by android_app.sh yet

# Test libs to run
matter.sdk.utest.lib=libPlatformTests.a